Automattic/pocket-casts-android
# Fix confirm button when backing out of payment flow

## Description

Makes the confirm button work after a user backs out of the Google payment flow.

I don't think this is the best fix here, but I think that the overall design of this fragment probably needs some love and I don't think this is the best time to tackle that. This fix is intended to be the minimal fix that addresses this issue without introducing any regressions.

Fixes #281

## Test

1. Log in with an account that does not have plus
1. From the account screen tap "Upgrade to Plus"
1. Select a frequency and tap "Next"
1. Agree to the TOS
1. Tap Confirm
1. When the bottom modal for payment from Google Play pops up, either tap above the modal or swipe back to dismiss it
1. Observe that tapping the "Confirm" button relaunches the Google Play payment flow

# Checklist
